src/runmode-af-packet.c

```diff
@@ -459,48 +497,63 @@ static void *ParseAFPConfig(const char *iface)
                 aconf->xdp_mode = XDP_FLAGS_DRV_MODE;
             } else if (!strcmp(xdp_mode, "hw")) {
                 aconf->xdp_mode = XDP_FLAGS_HW_MODE;
+                aconf->ebpf_t_config.flags |= EBPF_XDP_HW_MODE;
             } else {
                 SCLogWarning(SC_ERR_INVALID_VALUE,
                              "Invalid xdp-mode value: '%s'", xdp_mode);
             }
         }
+
+        if (ConfGetChildValueBoolWithDefault(if_root, if_default, "no-percpu-hash", (int *)&boolval) == 1) {
```

Comment:
should this be worded positively? 'use-percpu-hash'. Double negations confuse ppl.